Dockerfile

```diff
@@ -19,7 +19,29 @@ RUN ["sed", "-i", "s/exec \"$@\"/echo \"skipping $@\"/", "/entrypoint.sh"]
 RUN ["/entrypoint.sh", "--datadir", "/init-db"]
 
 
+
+### real container #################################################
+
 FROM mysql:5.7
 
 COPY --from=builder /init-db /var/lib/mysql
+COPY . /app
+
+ENV MYSQL_ALLOW_EMPTY_PASSWORD=yes
+ENV MYSQL_DATABASE=owid
+ENV DB_NAME=owid
+
+# Installing Node.js 12.x and Yarn
+RUN set -x \
+      && apt-get update && apt-get install -y curl jq \
+      && NODE_VERSION=$(jq -r .engines.node /app/package.json) \
+      && DEB_FILE="nodejs_${NODE_VERSION}-1nodesource1_amd64.deb" \
+      && curl -sLO "https://deb.nodesource.com/node_12.x/pool/main/n/nodejs/${DEB_FILE}" \
+      && apt-get install -y ./"${DEB_FILE}" && rm "${DEB_FILE}" \
+      && curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+      && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+      && apt-get update && apt-get install -y yarn \
+      && rm -rf /var/lib/apt/lists/*
+
+#RUN cd app && yarn
```

Comment:
Hi @abitrolly! I am using owid-grapher and planning on dockerizing - glad to see your PR.  Why is this line commented out?

Reply:
@masaez the line is commented out, because before running the app, there should be database running. I haven't found an example of running database in the same container. Examples recommend using `docker-compose` to start db in another container, but I wanted `owid-grapher` to be self-sufficient.

Reply:
I would also try to have the db in a separate containers (I am actually using the DB out of containers at all). Anyway, my comment was around the idea that running yarn would just resolve dependencies, not actually run the project. You would still need to run yarn dev or similar.

Reply:
As this PR got stalled, feel free to pick it to fix what is needed. I sent you an invite, so you should be able to commit to the fork.